arbitrum-docs/node-running/how-tos/read-sequencer-feed.md

```diff
@@ -8,7 +8,7 @@ todos:
   - Align on what we want to treat as proper nouns vs common nouns
 ---
 
-Running an Arbitrum relay locally as a Feed Relay lets you subscribe to the Sequencer feed for real-time data as the Sequencer accepts and orders transactions off-chain. 
+[Running an Arbitrum relay locally as a feed relay](/node-running/how-tos/running-a-feed-relay.mdx) lets you subscribe to the sequencer feed for real-time data as the sequencer accepts and orders transactions off-chain. 
```

Comment:
[Running an Arbitrum relay locally as a feed relay](/node-running/how-tos/running-a-feed-relay.mdx) lets you subscribe to an uncompressed sequencer feed for real-time data as the sequencer accepts and orders transactions off-chain.